src/vm/hints/hint_utils.rs

```diff
@@ -144,6 +143,24 @@ pub fn get_address_from_var_name(
     .ok_or(VirtualMachineError::FailedToGetIds)
 }
 
+//Gets the address of a variable name.
+//If the address is an MaybeRelocatable::Relocatable(Relocatable) return Relocatable
+//else raises Err
+pub fn get_relocatable_from_var_name(
+    var_name: &str,
+    ids: HashMap<String, BigInt>,
+    vm: &VirtualMachine,
+    hint_ap_tracking: Option<&ApTracking>,
+) -> Result<Relocatable, VirtualMachineError> {
+    if let MaybeRelocatable::RelocatableValue(relocatable) =
+        get_address_from_var_name(var_name, ids, vm, hint_ap_tracking)?
+    {
+        Ok(relocatable)
+    } else {
+        Err(VirtualMachineError::FailedToGetIds)
```

Comment:
```suggestion
        Err(VirtualMachineError::ExpectedRelocatable(...))
```

Reply:
Okay, I created this new VirtualMachineError
https://github.com/lambdaclass/cleopatra_cairo/pull/304/commits/2551d9564f13024bbd14c820c99cec547439bf79